ingestion/src/metadata/ingestion/source/ldap_users.py

```diff
@@ -85,9 +85,9 @@ def create(
     def next_record(self) -> Iterable[CreateUserRequest]:
```

Comment:
small detail, but we could change the signature here to return `Iterable[OMetaUserProfile]`